profiles/filters.py

```diff
@@ -0,0 +1,32 @@
+from django_filters import filters
+from django_filters.rest_framework import FilterSet
+
+
+class ProfileFilter(FilterSet):
+    is_saved = filters.BooleanFilter(method="is_saved_filter")
+    company = filters.BooleanFilter(field_name="is_registered")
+    startup = filters.BooleanFilter(field_name="is_startup")
+    activity_type = filters.CharFilter(field_name="activities__name")
+
+    def is_saved_filter(self, queryset, name, value):
+        if value:
+            if self.request.user.is_authenticated:
+                queryset = queryset.filter(saved_list__user=self.request.user)
+            else:
+                queryset = queryset.none()
+        return queryset
+
+    def company_filter(self, queryset, name, value):
+        if value:
+            return queryset.filter(is_registered=True)
+        return queryset
```

Comment:
Sorry for not mentioning it — when using `field_name`, defining a method is redundant. It should work without it.

Reply:
... and also, when filter name matches field name, you don't need even pass `field_name` argument.

```
is_startup = filters.BooleanFilter()
```
... and that's it. 

It also will be better this way, than a current implementation. As there will be no confusion between filter name and actual field name.

Reply:
I'll remove redundant methods. Is it obligatory to name filters as field names in model? Or my implementation is option too?